django_freeradius/base/admin.py

```diff
@@ -206,3 +208,21 @@ def get_readonly_fields(self, request, obj=None):
 
 
 AbstractRadiusBatchAdmin.list_display += ('expiration_date', 'created')
+
+
+class AbstractRadiusProfileAdmin(TimeStampedEditableAdmin):
+    pass
+
+
+class RadiusUserProfileInline(StackedInline):
```

Comment:
add this mixin from `openwisp-utils`: https://github.com/openwisp/openwisp-utils/blob/master/openwisp_utils/admin.py#L93